js/api-augment/definitions/capacity.ts

```diff
@@ -0,0 +1,31 @@
+export default {
+    rpc: {
+        dummy: { description: "This API has no custom RPCs", params: [], type: 'undefined' },
+    },
+    types: {
+        RewardEra: 'u32',
+        Balance: 'u128',
+        BlockNumber: 'u32',
+        UnclaimedRewardInfo: {
+            reward_era: 'RewardEra',
+            expires_at_block: 'BlockNumber',
+            staked_amount: 'Balance',
+            eligible_amount: 'Balance',
+            earned_amount: 'Balance',
+        }
+    },
+    runtime: {
```

Comment:
Nice. By the way, do this show up in PolkadotUI now?

Reply:
It won't until it merges to main and I think you still have to post a PR with the updated release to PolkadotJS ui